.npmrc

```diff
@@ -1,3 +1,4 @@
 # https://verdaccio.org/docs/authentication/
 
 registry=https://registry.dev:4873
+registry=https://registry.npmjs.org
```

Comment:
ref: https://stackoverflow.com/questions/32633678/is-there-any-way-to-configure-multiple-registries-in-a-single-npmrc-file

i think we should use local registry only for @diia-inhouse scope